models/classes/user/LtiUser.php

```diff
@@ -163,7 +174,7 @@ public function getPropertyValues($property)
                 $returnValue = array(DEFAULT_LANG);
                 break;
             case GenerisRdf::PROPERTY_USER_UILG :
-                $returnValue = array($this->language);
+                $returnValue = array($this->getLanguage());
```

Comment:
I'd suggest to use either `array()` or `[]` notation everywhere.
Also, each case can directly `return` the value (early exit):
```
public function getPropertyValues($property)
{
    switch ($property) {
        case GenerisRdf::PROPERTY_USER_DEFLG :
            return [DEFAULT_LANG];
        case GenerisRdf::PROPERTY_USER_UILG :
            return [$this->getLanguage()];
        case  GenerisRdf::PROPERTY_USER_ROLES :
            return $this->taoRoles;
        case  GenerisRdf::PROPERTY_USER_FIRSTNAME :
            return [$this->firstname];
        case  GenerisRdf::PROPERTY_USER_LASTNAME :
            return [$this->lastname];
        case  OntologyRdfs::RDFS_LABEL :
            return [$this->label];
    }
    
    \common_Logger::d('Unkown property ' . $property . ' requested from ' . __CLASS__);
    return [];
}
```
Even better, since all case do similar things, an array fits better here:
```
public function getPropertyValues($property)
{
    $properties = [
        GenerisRdf::PROPERTY_USER_DEFLG => [DEFAULT_LANG],
        GenerisRdf::PROPERTY_USER_UILG => [$this->getLanguage()],
        GenerisRdf::PROPERTY_USER_ROLES => $this->taoRoles,
        GenerisRdf::PROPERTY_USER_FIRSTNAME => [$this->firstname],
        GenerisRdf::PROPERTY_USER_LASTNAME => [$this->lastname],
        OntologyRdfs::RDFS_LABEL => [$this->label],
    ];

    if (isset($properties[$property])) {
        return $properties[$property];
    }
    
    \common_Logger::d('Unkown property ' . $property . ' requested from ' . __CLASS__);
    return [];
}
```
Only drawback for this third solution is that it executes `$this->getLanguage()` in all cases.

Reply:
Agree on the [] syntax over the array() syntax, but I did not want to change all occurrences with this PR, will change line 193 though. but I do not want to retrieve values that are never used and I try to prevent early exits (where possible) as there is a risk that a developer overlooks it when working with the code